portfolio/src/main/java/com/google/sps/servlets/SubmitEmailServlet.java

```diff
@@ -18,13 +31,49 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    // Get the value entered in the form.
-    String email = request.getParameter("email-address");
+    String email = Jsoup.clean(request.getParameter("email-address"), Whitelist.none());
+    long timestamp = System.currentTimeMillis();
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Email");
+    FullEntity taskEntity =
+        Entity.newBuilder(keyFactory.newKey())
+            .set("title", email)
+            .set("timestamp", timestamp)
+            .build();
+    datastore.put(taskEntity);
```

Comment:
wasn't this change reviewed in another PR https://github.com/elenapelaez/software-product-sprint/pull/4? Do you plan on submitting that PR as well

Reply:
I didn't know if I needed to add my new changes to Pull Request 4 or create a new Pull Request. I plan on submitting both